lib/php/admin-bundle/Controller/QueuesListController.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Alchemy\AdminBundle\Controller;
+
+use PhpAmqpLib\Connection\AMQPSSLConnection;
+use Symfony\Component\HttpFoundation\Response;
+use PhpAmqpLib\Connection\AMQPStreamConnection;
+use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+
+class QueuesListController extends AbstractController
+{
+    public function __construct()
+    {
+    }
+
+    #[Route(path: '/admin/queues/list', name: 'queues_list')]
+    public function __invoke(): Response
+    {
+        // TODO retrieve dynamically the queues
+        $Q = [
+            'databox' => ['p1', 'p2'],
+            'expose'  => ['p1'],
+            'uploader'=> ['p1', 'p2', 'p3']
+        ];
```

Comment:
Move it at least to bundle configuration (no service name should be mentionned in this bundle)